core/src/main/java/io/crate/types/ArrayType.java

```diff
@@ -168,4 +168,14 @@ public int hashCode() {
         result = 31 * result + innerType.hashCode();
         return result;
     }
+
+    public static double[] toDoubleArray(Object[] array) {
```

Comment:
since there is only one usage, and also no test for this method i propose to either inline it or make a private function where it is used